docs/installing-northstar/northstar-installers/flightcore-guide.md

```diff
@@ -0,0 +1,55 @@
+
+## Linux Installation 
+
+For Linux, you want to download `flight-core_X.Y.Z_amd64.AppImage` (X Y Z being the version numbers of FlightCore) from [FlightCore's GitHub releases page](https://github.com/R2NorthstarTools/FlightCore/releases), put it in a place you prefer, then make sure to right click on it, click on properties, and tick `Executable as Program` (as shown below), then you can just double click it to open FlightCore.
+
+![Make sure to tick this setting to be ON](../../images/flightcore-executable-as-program.png)
+
+## Main Menu
+
+![FlightCore Main Window](../../images/flightcore-main-window.png)
+
+This is the *Main Menu* of FlightCore. It includes the *Play*, *Changelog*, *Mods*, and *Settings* tabs, all of which will be covered in the guide.
+
+## Play 
+
+![Play tab's information](../../images/flightcore-main-window-information.png)
+
+The *Play* tab is actually the main window that FlightCore opens to. It includes some useful information in the bottom left, such as the version of Northstar you have installed, how many players are online, how many servers are online, and a button to launch Northstar.
+
+## Changelog
+
+![FlightCore's Changelog tab](../../images/flightcore-changelog.png)
+
+The *Changelog* tab displays information from Northstar's [GitHub page for releases](https://github.com/R2Northstar/Northstar/releases), detailing what a new update to Northstar brings.
+
+## Mods
+
+![FlightCore's Mods browser](../../images/flightcore-mod-browser-window.png)
+
+The *Mods* tab of Flightcore can display either mods installed locally, or a browser for installing mods from [Northstar's Thunderstore page](https://northstar.thunderstore.io/). The default option it opens to when clicking the tab is displaying the mods you have installed.
+
+`Local` displays mods you have installed currently, with options to toggle them on or off or delete them.
+
+`Online` displays the Mod browser, which sorts by newest uploads by default. Hitting install on a mod will automatically install it, and hitting the *i* next to it will open the mod's information page on ThunderStore in your browser.
+
+## Settings 
+
+![FlightCore's Settings tab](../../images/flightcore-settings.png)
+
+The *Settings* tab is the fourth and final tab of FlightCore. This tab allows you to view and choose your Titanfall2 directory (default directories can be found in the [Default Directories](../troubleshooting.md#game-location) section), select how many mods you want to see on in one page on the online browser in FlightCore, the version of FlightCore you're running, as well as a toggle to enable testing release channels.
+
+Note that your settings tab will most likely look different, as this screenshot is taken from a Linux machine, and also doesn't include owning Titanfall 2 on EA. Your installation folder should be wherever the default directory is, or wherever you moved the game to.
```

Comment:
Could probably skip that line cause the only difference you'll see between Linux and Windows is the path `C:\something\something` vs `/home/some_user/something/something` ^^